frontend/app/globals.css

```
@@ -0,0 +1,158 @@
+.panel {
+  backdrop-filter: blur(16px);
+  background: var(--panel);
+  border: 1px solid var(--panel-border);
+  border-radius: 28px;
+  box-shadow: 0 24px 80px rgba(0, 0, 0, 0.28);
+  padding: 1.75rem;
+}
+
+.hero h1 {
+  margin: 0.35rem 0 0.75rem;
+  font-family: "IBM Plex Mono", "SFMono-Regular", monospace;
```

Stylelint (17.7.0)

[error] Expected no quotes around "SFMono-Regular" (font-family-name-quotes)

(font-family-name-quotes)